edge/pkg/metamanager/client/service.go

```diff
@@ -70,20 +73,20 @@ func (s *services) GetPods(name string) ([]v1.Pod, error) {
 	}
 
 	if respMsg.GetOperation() == model.ResponseOperation {
-		return handlerServicePodListFromMetaDB(content)
+		return handleServicePodListFromMetaDB(content)
```

Comment:
Does this func still used? Or we can delete it.

Reply:
Since custom resource "backend" now has some trouble being sent to edge due to UID/ResourceVersion/... checking, "podlist" would be continue to be used.